data/templates/bootstrap.js

```diff
@@ -273,4 +273,22 @@ function getFile(downloadUrl, cb) {
     }).end();
 }
 
+/**
+ * Check valid accepted response code - check whether the code is an array of number
+ * @private
+ * @param code
+ */
+function checkValidAcceptCode(code) {
+    if (!(code instanceof Array)) {
+        return false;
+    }
+
+    return code.every(function(item) {
+        if (typeof item !== 'number') {
```

Comment:
This is fine, although if we wanted to be more flexible we could try to do parseInt() calls against each item, and only return false if that fails. That way `"127"` just becomes `127`